packages/react-i18n/src/i18n.ts

```diff
@@ -150,6 +155,36 @@ export default class I18n {
     return this.formatNumber(amount, {as: 'currency', ...options});
   }
 
+  unformatCurrency(input: string, currencyCode: string): string {
+    const nonDigits = /\D/g;
+    const decimal = this.decimalSymbol(currencyCode);
+    const lastDecimalIndex = input.lastIndexOf(decimal);
+
+    if (decimal === DECIMAL_NOT_SUPPORTED) {
+      const amount = input.replace(nonDigits, '');
+      return `${amount}.${DECIMAL_VALUE_FOR_CURRENCIES_WITHOUT_DECIMALS}`;
```

Comment:
Should we even return a decimal if they aren't supported by the currency?

Reply:
IMO, yes. The intention of `unformatCurrency` is to provide a standard representation of the formatted currency so it can then be used to send off a back end, or even back to`i18n.formatCurrency`. This is what we do in rails today. If a UI component formats a price to `12` in `JPY`, we store it as `12.00` in the backend.

Reply:
Oh I see. 
I would argue, why not returning a `number` instead of a `string` then? We would avoid some other problems, like the `toFixed(2)`?

Reply:
My understanding is that we store all our money inputs as strings. I believe `toFixed(2)` conversation has less to do with a string/number issue and more to do with knowing how many decimals places the consumer wants.